jib-core/src/test/java/com/google/cloud/tools/jib/api/JavaContainerBuilderTest.java

```diff
@@ -192,6 +193,21 @@ public void testToJibContainerBuilder_missingAndMultipleAdds()
     Assert.assertEquals(ImmutableList.of(), getExtractionPaths(buildConfiguration, "extra files"));
   }
 
+  @Test
+  public void testToJibContainerBuilder_setAppRootLate()
+      throws URISyntaxException, InvalidImageReferenceException, IOException {
+    try {
+      JavaContainerBuilder.fromDistroless()
+          .addClasses(getResource("core/application/classes"))
+          .setAppRoot("/oh no")
+          .toContainerBuilder();
```

Comment:
It's not clear who's throwing the exception. And `.toContainerBuilder()` isn't necessary, I think. Do

```java
try {
   containerBuilder.setAppRoot("/oh no");
```